microsite/core/Components.js

```diff
@@ -66,7 +66,7 @@ Block.Graphics = ({ padding, children }) => {
 };
 Block.Graphic = props => {
   /* Coordinates and size are in % of graphics container size, e.g. width={50} is 50% of parent width */
-  const { x = 0, y = 0, width = 0, src, className = '' } = props;
+  const { x, y, width, src, className = '' } = props;
```

Comment:
this makes it render `x: undefined%` etc in the actual css

Reply:
Yes . I did this because for each of the graphics we are supplying inline styles so if I put 0 initially & not provide inline values it will by default take 0% for each of those values & will not render the graphics . To apply styles using media queries we have to use either id or class selectors & for that I don't think there is a way to override inline styles using CSS selectors.It is rendering x:undefined% then it can be overridden by styles that I have defined in media queries . 
I found this solution working for this ..so implemented this. If it can be implemented in any other way . I'll try to check . and thanks for the review 😀 . @freben

Reply:
Yeah I understand the rationale, it's just that you have to form the CSS object differently below. The solution works now in the browser, but only because the CSS is "broken" because it's an illegal value so it gets ignored. Should be left out entirely instead, if it's undefined

Reply:
If I form CSS object differently then still the problem is to override the inline styles. @freben so should I change it ?